README.md

```diff
@@ -1,2 +1,35 @@
 # environ
-Package environ provides an easy way to store and manipulate environment state.
+
+Package environ provides an easy way to store and manipulate environment state. It is thread and concurrency safe.
+
+## Features
+
+The key features are:
+
+- FromOS, a function to inherit the current state.
+- The usual Get/Set/Unset functions.
+- Keep/Drop functions that work on lists of variables/prefix globs.
+- Concurrent safety.
+- Functions for reading the set in different forms. (slice, map)
+- Functions over the set of values (Len())
+- Globbing of keys to keep or drop whole collections of values.
```

Comment:
I think this is what threw me off to the globbing commentary below, I read globbing of keys and was thinking full glob patterns until reading the implementation (is this supposed to call out something of different than the keep/drop functions noted above which does explicitly call that out?). 

Perhaps update this to say like suffix globbing of keys to... (and to that, is it a actually prefix or suffix - I would have thought it implied what was globbed, so prefix glob would be *foo and suffix glob would bee foo*)